features/git-town/configuration/configuration.feature

```diff
@@ -9,28 +9,54 @@ Feature: listing the configuration
     Given I have configured the main branch name as "main"
     And my non-feature branches are "qa" and "staging"
     When I run `git town config`
-    Then I see "Main branch: main"
-    And I see "Non-feature branches: qa, staging"
+    Then I see
+      """
+      Main branch:
+      main
```

Comment:
I just don't know how to format this `git town config`. The main branch is only one branch but the non feature branches are a list.

Maybe this
```
$ git town config
Main branch:          [none]
Non feature branches: [none]

$ git town config
Main branch:          main
Non feature branches: qa
                      staging
```
I was also thinking of something that looked close to git branch output

```
$ git town config
M  main
NF qa
NF staging
```
But didn't know where to go if there were none